lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -251,6 +282,27 @@ export class ContentMetadataComponent implements OnChanges, OnInit, OnDestroy {
         }
     }
 
+    toggleEditMode(buttonType: ButtonType, group?: CardViewGroup) {
```

Comment:
I see you never pass group parameter to that function so this parameter can be removed

Reply:
This parameter allows us to specify which groups of panels should editable or uneditable when certain buttons are clicked.
if (group) {
                    group.editable = !group.editable;
                }

Reply:
@Yasa-Nataliya this is not addressed. As I said it looks like unused, please check all usages of that function. In every usage you pass only single parameter (buttonType) but you never pass group so it is unused as you never pass group to that function: 
<img width="652" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/6e792c2b-f7e9-4bcd-8fbc-6abadbc7f1dc">

Reply:
Now, we are passing the group parameter to the function, please check it.